.github/workflows/check-change.yml

```diff
@@ -1,8 +1,36 @@
-name: Check change
+name: Check changes/package versions
 on:
   pull_request:
 
 jobs:
+  check-package-versions:
+    runs-on: ubuntu-latest
+    env:
+      SYNCPACK_VERSION: latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
```

It's required in the other job for `beachball` to compare branches, but this job does not need to fetch all branches and tags 😉 

https://github.com/actions/checkout#Fetch-all-history-for-all-tags-and-branches

Reply:
right, I was a copy pasta dev guilty as charged